components/badge/badge_power.c

```diff
@@ -24,7 +24,7 @@ int
 badge_battery_volt_sense(void)
 {
 #ifdef ADC1_CHAN_VBAT_SENSE
-	int val = adc1_get_voltage(ADC1_CHAN_VBAT_SENSE);
+	int val = adc1_get_raw(ADC1_CHAN_VBAT_SENSE);
```

Comment:
could you make this a separate patch? "esp-idf update"

Reply:
The separate patch done as #215